src/cpuid.cpp

```diff
@@ -30,9 +30,6 @@ POSSIBILITY OF SUCH DAMAGE.
 
 */
 
-#include "libtorrent/config.hpp"
-#include <cstring>
-
 #include "libtorrent/aux_/cpuid.hpp"
 
 #if defined _MSC_VER && TORRENT_HAS_SSE
```

Comment:
Isn't the macro propagated via the include of `libtorrent/aux_/cpuid.hpp`? Do you prefer the explicit inclusion?